Spanner/tests/Snippet/TransactionTest.php

```diff
@@ -395,6 +395,25 @@ public function testCommit()
         $snippet->invoke();
     }
 
+    public function testGetCommitStats()
```

Comment:
This method no longer exists in the client. Remove the test and CI should pass.

Reply:
I'm confused. `getCommitStats` still exists and is needed for users to access `commitStats`. I've managed to get the test to pass however the CI is failing for seemingly unrelated reasons.